### ISSUE TITLE
Version updater didn't work

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#1654]
## Observed behaviour

When the version updater was run, the resulting nametable had nameID 5 as `0.024` 

## Expected behaviour

It should have been `Version 1.025`

## Resources and exact process needed to replicate

Please double check this script and make tests that prove it works the way it supposed to work :)

**Comment by felipesanches** [comment]
Which font file did you use to get this bad behaviour ?

**Comment by felipesanches** [comment]
beware that the helper scripts are all on gftools now, so this should be a bug reported in the https://github.com/googlefonts/tools/ issue tracker.

I'd suggest uninstalling old fontbakery scripts so that you don't end up using old versions of them by mistake.

**Comment by felipesanches** [comment]
There used to be a pair attributes on fontbakery-update-families.py to bump up the minor version number.

```
-b, --versionbump  increment font files' minor version number
```

but that was deprecated (see https://github.com/googlefonts/fontbakery/issues/1216) based on this discussion: https://github.com/googlefonts/fontbakery/commit/eb428fefde30613132ec65c3400aa21c6c7d498d#commitcomment-20713481

**Comment by felipesanches** [comment]
@davelab6 what is the version updater script you're referring to ?

**Comment by felipesanches** [comment]
I need feedback here, @davelab6

**Comment by chrissimpkins** [comment]
We implemented a method for the return of a tuple of major version, minor version integers in the `libfv` library based upon feedback from Felipe and this will be the basis for our major/minor version increment support in the library.  If it is helpful for this issue, here is a link to an example of the data that you get from the font binaries with either a file path string or a fonttools TTFont object:

https://github.com/source-foundry/font-v#work-with-majorminor-version-number-integers

I happened to wander across this issue report when I filed my PR.  If this is something that you would be willing to implement from the libfv project dependency and are interested in collaborating on the source to make it happen I am definitely interested and plan to support this in the library.
